modules/utils.py

```diff
@@ -226,7 +226,7 @@ def identify_build_environment(current_partition):
     else:
         system, partition = current_partition.fullname.split(':')
         cp_dir = os.path.realpath(
-            os.path.join(os.path.dirname(__file__), '..', 'spack-environments',
+            os.path.join(os.path.dirname(__file__), '..', 'envs',
```

Comment:
`pathlib` is more readable than `os.path`. So I'd change this to:
```
from pathlib import Path

cp_dir = Path(__file__).absolute().parent.parent / 'envs').
```

Then you can do on 232:
```
env_dir = cp_dir / partition
```